index.html

```diff
@@ -299,7 +656,7 @@ <h3>Deactivate</h3>
       <h2>Test Vectors</h2>
       <!-- No way to do directory listing on github pages short of generating them client side :-(  Not sure if there's a better way to do this -->
       <p>For a full list of test vectors see <a href="https://github.com/w3c-ccg/did-method-key/tree/master/test-vectors">test vectors</a>.</p>
-      
+
       <section class="informative">
         <h3>Ed25519 / X25519</h3>
```

Comment:
Same note as above: https://github.com/w3c-ccg/did-method-key/pull/51/files#r895671296

(I'm fine tackling this in another PR just trying to gage intent).

Reply:
Yeah, it's a good question, let's try to tackle answering that question in this PR.

I'm trying to be sensitive to the fact that IETF is saying that key derivation is fine (via the OSCORE work) and they're providing proofs (THORMAKER) to that effect, while FIPS is (frustratingly) not going to say that's ok for another decade or more. 

So, the best I think we can do is provide a flag that allows implementations to disable key derivation for Ed25519 -> X25519 in order to comply to FIPS audits.

Reply:
yeah agree, the citations help, let's just make sure the normative requirements are explicit enough to support either posture.

Reply:
Resolving to focus on remaining comments.